src/autoscaler/pruner/config/config_test.go

```diff
@@ -380,8 +372,8 @@ lock:
 				conf.Lock.ConsulClusterConfig = ""
 			})
 
-			It("should error", func() {
-				Expect(err).To(MatchError(MatchRegexp("Configuration error: Consul Cluster Config is empty")))
+			It("should validate successfully", func() {
+				Expect(err).NotTo(HaveOccurred())
```

Comment:
@kanekoh  We need test case to check whether config is valid even without consul_cluster_config. As partial test does not validate the config (only loads with default values), we need this test.